go/pkg/client/exec.go

```diff
@@ -255,6 +256,9 @@ func (c *Client) ExecuteAndWaitProgress(ctx context.Context, req *repb.ExecuteRe
 					progress(metadata)
 				}
 			}
+			if returnEarly {
+				return status.Error(codes.Internal, "fake error to for wait call")
```

Comment:
nit: "(...) error to for wait (...)", shouldn't it be, "(...) error to trigger the wait (...)"